core/public/images/slack-logo.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like you got the Slack icon from font awesome.  Is this file still neede?

Reply:
Nope! removed